tla-import/src/main/scala/at/forsyte/apalache/tla/imp/src/SaveToStoreTracker.scala

```diff
@@ -25,4 +25,34 @@ class SaveToStoreTracker(sourceStore: SourceStore) extends TransformationTracker
       }
       output
   }
+
+  /**
+    * Decorate a declaration transformation with a tracker.
+    *
+    * @param tr a declaration transformation
+    * @return a new declaration transformation that applies tr and tracks the changes
+    */
+  override def trackDecl(tr: TlaDeclTransformation): TlaDeclTransformation = {
+    input: TlaDecl =>
+      val output = tr(input)
+      if (output.ID != input.ID) {
+        sourceStore.findOrLog(input.ID).foreach { loc => sourceStore.add(output.ID, loc) }
```

Comment:
Looking at this particular line, it wasn't immediately obvious what `findOrLog` did, I would suggest changing the name to `findWithLogging`

Reply:
renamed to `findOrWarn`